### PR TITLE
chore: switch to pnpm instead of yarn

### LINKED ISSUE #1652: switch to pnpm as package manager
### Is your enhancement related to a problem? Please describe

See https://github.com/containers/podman-desktop/issues/8566

### Describe the solution you'd like

Use pnpm

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by jeffmaury**
Fixes #1652

### What does this PR do?

Switch package manager from yarn to pnpm

### Screenshot / video of UI

N/A

### What issues does this PR fix or reference?

#1652 

### How to test this PR?

Execute build

### CODE REVIEW

**Review by axel7083 — APPROVED** [review]
Need to rebase but LGTM

**Review by ScrewTSW — APPROVED** [review]

**Review by benoitf — APPROVED** [review]

**Review by benoitf — COMMENTED** [review]
Need to remove .yarnrc yaml file

**Comment by benoitf** [comment]
it's missing this file
https://github.com/containers/podman-desktop/blob/main/.npmrc

File: package.json [hunk 1]
```diff
@@ -53,6 +54,7 @@
     "globals": "^15.9.0",
     "prettier": "^3.3.3",
     "prettier-plugin-svelte": "^3.2.6",
+    "svelte": "5.0.0-next.243",
```
Comment (benoitf) [inline package.json:57]: you should remove all new dev dependencies from there

I think the main issue is the missing .npmrc with the hoisted parameter so tooling will work properly

Comment (jeffmaury) [reply package.json:57]: I added svelte here so that eslint uses Svelte 5 instead of Svelte 4 but will check

**Comment by benoitf** [comment]
Looks like you also need to provide the pnpm file with the workspace definition

**Comment by benoitf** [comment]
https://raw.githubusercontent.com/containers/podman-desktop/main/pnpm-workspace.yaml

File: package.json (hunk 1 above)
Comment (benoitf) [reply package.json:57]: It was also missing the workspaces this is also why it was failing
There should be the same dependencies

**Comment by benoitf** [comment]
@jeffmaury if you could update the lock file

**Comment by jeffmaury** [comment]
See https://github.com/postmanlabs/postman-code-generators/issues/760

**Review by benoitf — COMMENTED** [review]
it seems new dependencies in the root package.json are not needed

(svelte and @tsconfig/svelte)

**Comment by ScrewTSW** [comment]
@jeffmaury I think `tests/playwright` are missing `pnpm-lock.yaml`, or is it all handled in the root project?

**Comment by benoitf** [comment]
@ScrewTSW there is only one lock file per project (it is handling all workspaces defined in pnpm-workspace file)

**Review by ScrewTSW — APPROVED** [review]
LGTM

**Pull request merged**